react/components/Tabs/README.md

```diff
@@ -1,3 +1,18 @@
+### Overview
+Tabs are navigation solutions for alternating between content that is in the same level of hierarchy.
+
+### Dos
+- Use short labels.
+- Use tabs in any place of your page hierarchy: as a top-level navigation or inside another component.
+- Use may use icons instead of text if you want, but be mindful it may hurt usability.
+
+### Don'ts
+- Don't use too many tabs. More than 5 might start looking weird.
+- Don't use dropdown menus with these tabs. I mean, one day we might support that, but we don't .
```

Comment:
Cuidado, tem uso de "We" & "I" nas docs. Melhor ficar com "uma pessoa falando". Acho que "We" seria melhor.